examples/api/lib/services/binding/handle_request_app_exit.0.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for [ApplicationLifecycleListener].
```

Comment:
nit: update this comment since ApplicationLifecycleListener is not a thing for now.